ship/src/test/scala/ch/epfl/bluebrain/nexus/ship/S3RunShipSuite.scala

```diff
@@ -1,78 +1,51 @@
 package ch.epfl.bluebrain.nexus.ship
 
-import cats.effect.IO
-import cats.syntax.all._
 import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.operations.s3.LocalStackS3StorageClient
-import ch.epfl.bluebrain.nexus.ship.RunShipSuite.{clearDB, expectedImportReport}
-import ch.epfl.bluebrain.nexus.ship.S3RunShipSuite.uploadFileToS3
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.operations.s3.LocalStackS3StorageClient.uploadFileToS3
+import ch.epfl.bluebrain.nexus.delta.sourcing.offset.Offset
+import ch.epfl.bluebrain.nexus.delta.sourcing.postgres.Doobie
+import ch.epfl.bluebrain.nexus.ship.RunShipSuite.expectedImportReport
+import ch.epfl.bluebrain.nexus.ship.config.ShipConfigFixtures
 import ch.epfl.bluebrain.nexus.testkit.mu.NexusSuite
 import eu.timepit.refined.types.string.NonEmptyString
 import fs2.aws.s3.models.Models.BucketName
 import fs2.io.file.Path
-import io.laserdisc.pure.s3.tagless.S3AsyncClientOp
 import munit.AnyFixture
-import org.scalatest.time.SpanSugar.convertIntToGrainOfTime
-import software.amazon.awssdk.services.s3.model.{CreateBucketRequest, PutObjectRequest, PutObjectResponse}
 
-import java.nio.file.Paths
-import scala.concurrent.duration.Duration
+import scala.concurrent.duration.{Duration, DurationInt}
 
-class S3RunShipSuite extends NexusSuite with RunShipSuite.Fixture with LocalStackS3StorageClient.Fixture {
+class S3RunShipSuite
```

Comment:
In a later PR, we could change this test to limit it to read the file and not run the import itself